### DISCUSSION TITLE
Add a function for getting/setting all control rods of one color

### DISCUSSION THREAD

**Discussion by LukeTech2006** [m-doescode/cchbm#1] [Features, Suggestions, & Ideas]
e.g.
setRods(50, colors.red) to modify Red Group
setRods(50) / setRods(50, -1) to modify All Groups

**Comment by m-doescode** [comment]
Sorry can you clarify exactly which type of reactor this is for? (After reading online, I assume this is for the RBMK, can you confirm?)

Also, if this is about RBMKs, I'm not exactly familiar with them, so do you mind briefing me about how they work and how the rods are controlled? Thanks.

**Reply by LukeTech2006** [reply]
Yes, I do mean RBMK Reactors.
As for controlling them, I can send you the links to the respective *.java files for the controller and stuff; I don't really know java :D

**Reply by m-doescode** [reply]
The source files are not a worry, but how would the rods be connected to the computer? Is there a remote controller block for RBMKs built into HBM? Or should I make my own linking system that way they can be linked and controlled via the peripheral API?

I'm asking because I'm not exactly familiar with RBMKs so I'm unsure of the most optimal way to connect them up to computers.

**Reply by LukeTech2006** [reply]
Best way would be to implement the RBMK Control Console as a peripheral, I think.
(As thats where you connect the reactor to)

**Reply by m-doescode** [reply]
Okay, I'll check that out! Thank you

**Comment by m-doescode** [comment]
[Done!](https://github.com/m-doescode/cchbm/releases/tag/v0.2)
